default/stringtables/en.txt

```diff
@@ -8679,14 +8685,38 @@ SHP_DEATH_SPORE
 Death Spores
 
 SHP_DEATH_SPORE_DESC
-Unlocks the [[shippart SP_DEATH_SPORE]] which allows ships to reduce the population of enemy planets.
+Unlocks the [[shippart SP_DEATH_SPORE]] which allows ships to reduce the Organic population of enemy planets.
```

Comment:
Ah. You haven't reverted the death spores/bio terminators back to organic only hence my partial confusion earlier.